config/base/rbac/role.yaml

```diff
@@ -20,12 +20,6 @@ rules:
   - olm.operatorframework.io
   resources:
   - catalogmetadata
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - olm.operatorframework.io
-  resources:
```

Comment:
Why?

Reply:
As long as we're sure it was done automatically, that's fine.

Reply:
Ah… it appears to be combining `catalogmetadata` and `clustercatalog` into one “stanza”, since they share properties.
But `diff` don't make it easy